### PR TITLE
Cr validation

### PR DESCRIPTION
**Pull request by sanchezl**

### CODE REVIEW

File: pkg/admission/customresourcevalidation/customresourcevalidator.go [hunk 1]
```diff
@@ -91,14 +91,12 @@ func (a *validateCustomResource) Validate(uncastAttributes admission.Attributes)
...
-			admission.NewForbidden(attributes, fmt.Errorf("unhandled subresource: %v", attributes.GetSubresource()))
+			return admission.NewForbidden(attributes, fmt.Errorf("unhandled subresource: %v", attributes.GetSubresource()))
 		}
 
 	default:
-		admission.NewForbidden(attributes, fmt.Errorf("unhandled operation: %v", attributes.GetOperation()))
+		return admission.NewForbidden(attributes, fmt.Errorf("unhandled operation: %v", attributes.GetOperation()))
 	}
-
-	return nil
```
Comment (deads2k) [inline pkg/admission/customresourcevalidation/customresourcevalidator.go]: thanks

File: pkg/admission/customresourcevalidation/customresourcevalidator_test.go [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/apiserver/pkg/admission"
+	"k8s.io/apiserver/pkg/authentication/user"
+	"k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	configv1 "github.com/openshift/api/config/v1"
+)
+
+const (
+	testGroup    = "config.openshift.io"
```
Comment (deads2k) [inline pkg/admission/customresourcevalidation/customresourcevalidator_test.go]: local to your test case

File: pkg/admission/customresourcevalidation/customresourcevalidator_test.go [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+	testVersion  = "v1"
+	testResource = "images"
+	testKind     = "Image"
+)
+
+var (
+	testObjectType *configv1.Image
```
Comment (deads2k) [inline pkg/admission/customresourcevalidation/customresourcevalidator_test.go]: local to test case.

File: pkg/admission/customresourcevalidation/customresourcevalidator_test.go (hunk 2 above)
Comment (sanchezl) [reply pkg/admission/customresourcevalidation/customresourcevalidator_test.go]: Done

File: pkg/admission/customresourcevalidation/customresourcevalidator_test.go (hunk 3 above)
Comment (sanchezl) [reply pkg/admission/customresourcevalidation/customresourcevalidator_test.go]: Done

**Pull request merged**